### ISSUE TITLE
how to assemble for orange pi 5 plus (processor - pk3588)

### ISSUE THREAD

**Issue by InfernalWolf666** [nillerusr/source-engine#330]
My operating system is Ubuntu 22.04 from Joshua Riek.

Please give me step-by-step instructions on how to remove it.

The instructions in the description are not accurate. I don't understand .

**Comment by ArtemS2** [comment]
My instructions are certainly not perfect, but you can try. :)

Step 1

Install required dependencies.

sudo apt install libsdl2-dev libfreetype6-dev libfontconfig1-dev libopenal-dev libjpeg-dev libpng-dev libcurl4-gnutls-dev libbz2-dev libedit-dev

Step 2

Copy the command repository:

git clone --recursive --depth 1 https://github.com/nillerusr/source-engine.git

Step 3

Configure the assembly

CXX=clang++ ./waf configure -T release 

Step 4 

Build source

./waf build

Step 5 

Saving ready-made sources

 ./waf install --strip --destdir=./sourse-engine_arm64
 
 (The folder "source-engine arm64" appears in the folder of downloaded sources, which you can then transfer to the game resources)
 
Step 6

Combine the collected sources and game resource files

**Comment by ferrellsl** [comment]
Yeah, this has been done nearly a year ago.  See:  https://github.com/Joshua-Riek/ubuntu-rockchip/discussions/74

**Comment by ferrellsl** [comment]
> @ferrellsl What other interesting games have you managed to collect?

I've also managed to build Return to Castle Wolfenstein, Quake 2 and Doom III.  Instructions to build all of these are over at:  https://github.com/Joshua-Riek/ubuntu-rockchip/discussions

**Comment by ferrellsl** [comment]
Oh,  I forgot,  I also managed to build EDuke 32 for Duke Nulem fans.

**Comment by ArtemS2** [comment]
@ferrellsl  I managed to run: :)
1) Doom 1
2) Doom 2
3) Final Doom
4) DOOM 3
5) DOOM 3 Resurrection of Evil
6) Quake 1
7) Quake 2
8) Quake 3
9) Half life
10) Half life Blue Shift
11) Half life Opposing Force
12) Half life Deathmatch Classic
13) Conter Strike 1.6
14) Half life 2
15) Half life episode 1
16) Half life episode 2
17) Half life 2 loastcost
18) Half life 2 deathmatch
19) Half life source
20) Portal
21) Conter Strike Source
22) Day of Defeat Source
23) Serious Sam 1
24) Duke Nukem 3d
25) Diablo
26) Wangers
27) Wolfenstein Enemy Territory
28) Wolfenstein 3D
29) Return to Castle Wolfenstein
30) Minecraft
31) Space Pinball
32) Cube 2
33) Red Eclipse 2
34) GTA 3
35) GTA Vice City
36) Heroes of Might and Magic III
37) Aliens VS Predator
38) Star Wars Jedi Academy
39) Star Wars Jedi Knight II: Jedi Outcast (Multiplayer)
40) The Elder Scrolls 3 Morrowind
41) Fallout
42) Fallout 2
43) Megaglest
44) Red Alert (demo version works, I haven’t tested the full version yet)
45) Nanosaur 2
46) Aquaria
47) Mario
48) Unreal Tournament

**Comment by ferrellsl** [comment]
Good for you.  I have a day job here:  https://lidarwidgets.com/ so I have to focus my time on software that pays the bills.  Games programming is just a hobby and is a nice break from the pressure of programming for a living.

**Comment by InfernalWolf666** [comment]
thanks friends. History will not forget you
